src/routes/github/github-router.ts

```diff
@@ -15,13 +17,13 @@ GithubRouter.use(GithubOAuthRouter);
 // CSRF Protection Middleware for all following routes
 GithubRouter.use(csrfMiddleware);
 
-GithubRouter.post("/setup", GithubSetupPost);
+GithubRouter.route("/setup")
+	.get(query("installation_id").isInt(), returnOnValidationError, GithubSetupGet)
+	.post(GithubSetupPost);
```

Comment:
Don't need github oauth flow for this anymore as we're not using a user API.  We are checking to make sure `installation_id` is set in querystring though.